index.bs

```diff
@@ -525,11 +533,14 @@ Note: At time of writing, <a spec="storage">obtain a storage bottle map</a> can
 but this will be refactored to support [=service workers=] which attempt to access storage on every navigation, and thus is not considered
 when updating the [=bounce tracking record/storage access set=].
 
-1. Let |origin| be |environment|'s [=environment/top-level origin=].
-1. If |origin| is null or an [=opaque origin=], then abort these steps.
 1. Let |browsingContext| be |environment|'s [=environment/target browsing context=].
 1. If |browsingContext| is null, then abort these steps.
+1. Let |activeDocument| be |browsingContext|'s active document.
+1. Let |origin| be |activeDocument|'s [=Document/origin=].
 1. Let |topLevelTraversable| be |browsingContext|'s [=browsing context/top-level traversable=].
+1. If |activeDocument|'s node navigable's [=navigable/parent=] is non-null:
+    1. Let |firstPartyOrigin| be |topLevelTraversable|'s [=navigable/active document=]'s [=Document/origin=].
+    1. If |origin| and |firstPartyOrigin| are not [=same site=], then abort these steps.
```

Comment:
Hmm, I actually think the original algorithm was correct.

We want to consider both same-site and cross-site iframes.  Yes, cross-site iframes will get partitioned storage, but that partitioned storage can effectively be used the same as a first-party cookie.  Since bounce tracking cares about 1P cookies we should treat it the same.

So that also means we want the `|origin|` to be associated with the top-level frame.  (Because any storage under the top-level will be partitioned by the top-level site.  We want to associate all those storage accesses with the top-level storage site.)

Reply:
Done, reverted the storage algorithm.